CONTRIBUTING.md

```diff
@@ -388,6 +388,30 @@ it('should open modal when button is clicked', () => {
 });
 ```
 
+**Example (Portal and Wrapper Test)**
+
+```jsx
+it('Tooltip with Portal', () => {
```

Comment:
Make sure to align the code formatting with the existing examples. For instance, add lines between code blocks, use `const` for the `Component` variable instead of `let` since its value has not changed, and add curly braces for the `provider` arrow function.